microcosm_elasticsearch/indexing.py

```diff
@@ -0,0 +1,29 @@
+"""
+Index management.
```

Comment:
I still need to research our and the communities migration approaches; I'm hoping to grow this module into a proper set of tools for managing indexes, including migrations.